src/cli/README.md

```diff
@@ -1169,6 +1169,27 @@ DNS response for example.com. - fd4c:9574:3720:2:0:0:5db8:d822 TTL:20456
 Done
 ```
 
+### dns resolve4 \<hostname\> \[DNS server IP\] \[DNS server port\] \[response timeout (ms)\] \[max tx attempts\] \[recursion desired (boolean)\]
+
+Send DNS query to obtain IPv4 address for given hostname.
+
+Requires `OPENTHREAD_CONFIG_DNS_CLIENT_NAT64_ENABLE`.
+
+The parameters after `hostname` are optional. Any unspecified (or zero) value for these optional parameters is replaced by the value from the current default config (`dns config`).
+
+This command requires a NAT64 prefix to be configured and present in Thread Network Data.
+
+The outputted addresses are the NAT64 IPv6 translated versions of the IPv4 addresses from the query response.
+
+```bash
+> route
+2001:db8:122:344::/64 sn med d800
+Done
```

Comment:
Is it necessary to explicitly configure this? Shouldn't OTBR configure this automatically?

Reply:
My idea was to show that we have a NAT64 prefix already (but used the `route` command which shows local netdata). 

I removed it in new push. Added text to mention the NAT64 prefix.